### DISCUSSION TITLE
Python pour la data science - Sélection de variables : une introduction

### DISCUSSION THREAD

**Discussion by giscus** [linogaliana/python-datascientist#463] [General]
# Python pour la data science - Sélection de variables : une introduction

L’accès à des bases de données de plus en plus riches permet des modélisations de plus en plus raffinées. Cependant, les modèles parcimonieux sont généralement préférables aux modèles extrêmement riches pour obtenir de bonnes performances sur un nouveau jeu de données (prédictions out-of-sample). Les méthodes de sélection de variables, notamment le LASSO, permettent de sélectionner le signal le plus pertinent dilué au milieu du bruit lorsqu’on a beaucoup d’information à traiter.

https://pythonds.linogaliana.fr/content/modelisation/4_featureselection.html

**Comment by daniel-odc** [comment]
hello Lino, dans le hint de la partie "Pénalisation", ça n'est pas très clair à quoi \lambda se réfère.
